auto_install/install.sh

```diff
@@ -947,8 +947,8 @@ askClientDNS(){
 			pivpnDNS1="$pivpnDNS2"
 			unset pivpnDNS2
 		elif [ -z "$pivpnDNS1" ] && [ -z "$pivpnDNS2" ]; then
-			pivpnDNS1="8.8.8.8"
-			pivpnDNS2="8.8.4.4"
+			pivpnDNS1="9.9.9.9"
+			pivpnDNS2="149.112.112.112"
 			echo "::: No DNS provider specified, using Google DNS ($pivpnDNS1 $pivpnDNS2)"
```

Comment:
You should update this line too.

Reply:
Thanks @orazioedoardo! I'm not sure why these didn't show up in my search.